### ISSUE TITLE
False/Positive static closure

### ISSUE THREAD

**Issue by fruitl00p** [kalessil/phpinspectionsea#1309]
When a method in a class 'wraps' a call to parent the closure can't be static:
```php
<?php

class A
{
    public function test()
    {
        return 'A';
    }
}

class B extends A
{
    public function test()
    {
        return (function ()
        {
            return parent::test();
        })();
    }
}
```
Thus the inspection probably checks for $this in the closure, but should also check for `parent::` :)

**Comment by kalessil** [comment]
oops, thank you for reporting @fruitl00p !

**Comment by kalessil** [comment]
Fixed!
